nucypher/characters/lawful.py

```diff
@@ -516,23 +514,24 @@ def __init__(self,
             # Ephemeral Self-Ursula
             #
             if is_me:
-                self.suspicious_activities_witnessed = {'vladimirs': list(), 'bad_treasure_maps': list()}
+                self.suspicious_activities_witnessed = {'vladimirs': [], 'bad_treasure_maps': []}
```

Comment:
:1st_place_medal:

Reply:
That's what we call a "merge commit" with "conflict resolution" ;-)